Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -177,5 +182,26 @@ void OnKeyboardBackPressed(object sender, EventArgs eventArgs)
 			ElementController?.SendCompleted();
 			Control?.ClearFocus();
 		}
+
+		void UpdateMaxLength()
+		{
+			var currentFilters = new List<IInputFilter>(Control?.GetFilters() ?? new IInputFilter[0]);
+
+			for (var i = 0; i < currentFilters.Count; i++)
+			{
+				if (currentFilters[i] is InputFilterLengthFilter)
+				{
+					currentFilters.RemoveAt(i);
+					break;
+				}
+			}
+
+			currentFilters.Add(new InputFilterLengthFilter(Element.MaxLength));
+
+			Control?.SetFilters(currentFilters.ToArray());
+
+			if (Control?.Text.Length > Element.MaxLength)
```

Comment:
You fetch Control.Text 3 times here. This causes 3 GetValue calls to be sent down to BindableObject. In this case it should be trivial to use a temporary variable to avoid that :)